### ISSUE TITLE
Update `@yamada-ui/tabs` accessibility

### ISSUE THREAD

**Issue by nakayan5** [yamada-ui/yamada-ui#1636]
### Description

his issue aims to address accessibility deficiencies in our tabs component as per W3C guidelines.
The component currently does not meet the accessibility standards outlined in the W3C documentation.

The goal is to ensure the component is accessible to all users, including those using assistive technologies.
Ensure that all unchecked items in this checklist should be addressed.

- [x] tab
  - [x] When the tab list contains the focus, moves focus to the next element in the page tab sequence outside the tablist, which is the tabpanel unless the first element containing meaningful content inside the tabpanel is focusable.

- [x] When focus is on a tab in a tablist with either horizontal or vertical orientation:
  - [x] Space or Enter: Activates the tab if it was not activated automatically on focus.

WAI-ARIA Roles, States, and Properties

- [x] If the tab list has a visible label, the element with role tablist has [aria-labelledby](https://w3c.github.io/aria/#aria-labelledby) set to a value that refers to the labelling element. Otherwise, the tablist element has a label provided by [aria-label](https://w3c.github.io/aria/#aria-label).
- [ ] Each element with role tab has the property [aria-controls](https://w3c.github.io/aria/#aria-controls) referring to its associated tabpanel element.
- [x] The active tab element has the state [aria-selected](https://w3c.github.io/aria/#aria-selected) set to true and all other tab elements have it set to false.
- [ ] Each element with role tabpanel has the property [aria-labelledby](https://w3c.github.io/aria/#aria-labelledby) referring to its associated tab element.
- [x] If the tablist element is vertically oriented, it has the property [aria-orientation](https://w3c.github.io/aria/#aria-orientation) set to vertical. The default value of aria-orientation for a tablist element is horizontal.

### Problem Statement/Justification

none

### Alternatives

none

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

**Comment by uraaaa24** [comment]
> The active tab element has the state [aria-selected](https://w3c.github.io/aria/#aria-selected) set to true and all other tab elements have it set to false.

Already supported.
https://github.com/yamada-ui/yamada-ui/blob/main/packages/components/tabs/src/tab.tsx#L79

**Comment by uraaaa24** [comment]
>  tab
 When the tab list contains the focus, moves focus to the next element in the page tab sequence outside the tablist, which is the tabpanel unless the first element containing meaningful content inside the tabpanel is focusable.

Already supported.
This can be achieved by setting `isManual` to true.
https://github.com/yamada-ui/yamada-ui/blob/main/packages/components/tabs/src/tab.tsx#L44
https://yamada-ui.com/components/disclosure/tabs#manual-activation

**Comment by uraaaa24** [comment]
> tab
 When the tab list contains the focus, moves focus to the next element in the page tab sequence outside the tablist, which is the tabpanel unless the first element containing meaningful content inside the tabpanel is focusable.

Already supported.
![2024-09-239 38 26-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/c091a0a4-9a7c-43c3-872b-cee56e4024b1)

**Comment by uraaaa24** [comment]
> If the tablist element is vertically oriented, it has the property [aria-orientation](https://w3c.github.io/aria/#aria-orientation) set to vertical. The default value of aria-orientation for a tablist element is horizontal.

This pull request will be handled by this pull request.
https://github.com/yamada-ui/yamada-ui/pull/2962

**Comment by uraaaa24** [comment]
Sorry, but I don't think I'll be able to fill in the rest of the checklist anytime soon, so I'll remove my assignment.
